src/app/services/guess-word.service.spec.ts

```diff
@@ -1,12 +1,15 @@
 import { TestBed } from '@angular/core/testing';
 
 import { GuessWordService } from './guess-word.service';
+import { HttpClient, HttpHandler } from '@angular/common/http';
 
 describe('GuessWordService', () => {
   let service: GuessWordService;
 
   beforeEach(() => {
-    TestBed.configureTestingModule({});
+    TestBed.configureTestingModule({
+      providers: [HttpClient, HttpHandler],
```

Comment:
```suggestion
      imports: [HttpClientTestingModule],
```

When u are testing the http service, u need to import the HttpTestingModule in imports